app/views/layouts/_social_meta.html.haml

```diff
@@ -1,5 +1,5 @@
-%meta{name: 'title', content: 'hackEDU'}
+%meta{name: 'title', content: 'Hack Club'}
 %meta{name: 'description', content: 'Be a part of the global movement of free student-led coding clubs.'}
 %meta{name: 'og:type', content: "website"}
-%meta{name: 'og:url', content: "https://hackedu.us"}
+%meta{name: 'og:url', content: "https://hackclub.io"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace